tests/functional/table-create-error/expected-stderr

```diff
@@ -0,0 +1 @@
+Failed to create table using sql "CREATE  TABLE `users-test_name` (`id` int) STORED AS orc": [MapR][Hardy] (35) Error from server: error code: '1' error message: 'org.apache.hive.service.cli.HiveSQLException: Error while processing statement: FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask. org.apache.hadoop.hive.ql.metadata.HiveException: [users-test_name]: is not a valid table n'. S1000
```

Comment:
Pridal som na to aj test, ktory sa spusta iba oproti `Hive DB 2.3.X` kedze sa chybove spravy medzi verziami mierne lisia.